### ISSUE TITLE
[release-1.21] Wrong images in `rke2-images-multus.linux-amd64.txt`

### ISSUE THREAD

**Issue by bk201** [rancher/rke2#2229]
**Environmental Info:**
RKE2 Version: `v1.21.7+rke2r1`

**Describe the bug:**

Multus images have been bumped to newer versions:
https://github.com/rancher/rke2-charts/blob/28ac3605e54d09e6e52549deffb07d3bbad34a04/packages/rke2-multus/charts/values.yaml#L1-L16
```
multus:
  image:
    repository: rancher/hardened-multus-cni
    tag: v3.7.1-build20211119

cniplugins:
  image:
    repository: rancher/hardened-cni-plugins
    tag: v0.9.1-build20211119
```

But the [published images list](https://github.com/rancher/rke2/releases/download/v1.21.7%2Brke2r1/rke2-images-multus.linux-amd64.txt) file `rke2-images-multus.linux-amd64.txt` of `v1.21.7+rke2r1` still has older ones:
```
docker.io/rancher/hardened-multus-cni:v3.7.1-build20210416
docker.io/rancher/hardened-cni-plugins:v0.9.1-build20210414
```

Harvester relies on the images list file to fetch images for air-gapped environments.

**Comment by guangbochen** [comment]
this is not a blocker issue but would prefer to get it fixed AEAP so we don't need to add those desired images manually for its future version, thanks.

**Comment by brandond** [comment]
You're right. I bumped the multus chart but missed those two images in the build script. I'd intended to work with @manuelbuil to get the rest of the images rebuild before bumping that chart in RKE2.

https://github.com/rancher/rke2/commit/e5b68727e27e9c0755e5cfb0ef4366de13120e0e

**Comment by brandond** [comment]
Please don't close this, we need to fix it on our side.

**Comment by guangbochen** [comment]
> Please don't close this, we need to fix it on our side.

I see it is because of keyword `Fix rancher/rke2#2229`, cc @bk201 prefer to use the `address` or `resolve` next time in the PR commits, thanks.

**Comment by bk201** [comment]
> > Please don't close this, we need to fix it on our side.
> 
> I see it is because of the keyword `Fix rancher/rke2#2229`, cc @bk201 prefer to use the `address` or `resolve` next time in the PR commits, thanks.

My bad, I didn't expect a comment in harvester-installer closes a RKE2 issue.

**Comment by mdrahman-suse** [comment]
Validated can successfully use multus during airgap install with v1.21.8-rc1+rke2r1 via private registry and via tarball. Confirmed it is using `rancher/hardened-multus-cni:v3.7.1-build20211119`
